bentoml/_internal/bento/bento.py

```diff
@@ -80,7 +94,7 @@ def _Bento_set_fs(bento: "Bento", _: t.Any, new_fs: "FS") -> "FS":
 
 
 @attr.define(repr=False, auto_attribs=False)
-class Bento(StoreItem):
+class Bento(RemoteStoreItem):
```

Comment:
We should probably not do this - `bentoml.get` returns a `Bento` which means a local Bento, and naming it RemoteStoreItem doesn't seem right